src/wincng.c

```diff
@@ -236,6 +243,82 @@
 #define PKCS_RSA_PRIVATE_KEY ((LPCSTR)(size_t)43)
 #endif
 
+static int
+_libssh2_wincng_bignum_resize(_libssh2_bn* bn, ULONG length);
+
+
+/*******************************************************************/
+/*
+ * Windows CNG backend: ECDSA-specific declarations.
+ */
+#if LIBSSH2_ECDSA
+
+typedef enum {
+    WINCNG_ECC_KEYTYPE_ECDSA = 0,
+    WINCNG_ECC_KEYTYPE_ECDH = 1,
+} _libssh2_wincng_ecc_keytype;
+
+typedef struct __libssh2_wincng_ecdsa_algorithm {
+    /* Algorithm name */
+    const char *name;
+
+    /* Key length, in bits */
+    ULONG key_length;
+
+    /* Length of each point, in bytes */
+    ULONG point_length;
+
+    /* Name of CNG algorithm provider, indexed by _libssh2_wincng_ecc_keytype */
```

Comment:
checksrc complains about long lines and whitespace:
https://github.com/libssh2/libssh2/actions/runs/7674923441/job/20929010381?pr=1315#step:3:5

Reply:
I fixed some of the warnings, but I'm going to need at least one more round.

Reply:
All `checksrc` violations should be fixed now